src/styles/01-colors/_colors-utility.scss

```diff
@@ -0,0 +1,66 @@
+/** Utility Colors */
+
+// SASS Color Variables
+
+$black: black;
+$white: white;
+
+// Grayscale
+$gray-xdark: #424242;
+$gray-dark: #616161;
+$gray-medium: #bdbdbd;
+$gray-light: #e0e0e0;
+$gray-warm-light: #efedeb;
+$gray-xlight: #f5f5f5;
+$gray-warm-xlight: #f8f8f7;
+$gray-xxlight: #fafafa;
+
+// State and link utilities
+$success: #077719;
+$error: #97272c;
+$focus: #4181f1;
+$link-primary: #0576d3;
+$link-secondary: #004b98;
+
+//Mapping where we associate a name string with each SASS var
+$colors: (
+  black: $black,
+  white: $white,
+  gray-xdark: $gray-xdark,
+  gray-dark: $gray-dark,
+  gray-medium: $gray-medium,
+  gray-light: $gray-light,
+  gray-warm-light: $gray-warm-light,
+  gray-xlight: $gray-xlight,
+  gray-warm-xlight: $gray-warm-xlight,
+  gray-xxlight: $gray-xxlight,
+  success: $success,
+  error: $error,
+  focus: $focus,
+  link-primary: $link-primary,
+  link-secondary: $link-secondary,
+);
+
+//generates a series of color classes that can be used anywhere
+@mixin color-classes {
+  @each $name, $color in $colors {
+    .cl-#{$name} {
+      color: $color;
+    }
+  }
+}
+
+//generates a collection of CSS vars
+@mixin css-vars {
+  @each $name, $color in $colors {
+    --ui-#{$name}: #{$color};
+  }
+}
+
```

Comment:
Looks like you can have a utility scss file for these mixins so you don't have to declare them again in this file. Also, it'd be good to have this in a separate PR next time, just easier to see all the related changes

Reply:
Done!